### PR TITLE
Circle CI 2.0 update

### PR DESCRIPTION
**Pull request by kruszczynski**
🆙

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  test:
+    docker:
+      - image: circleci/ruby:2.4.2
+    steps:
+      - checkout
+      - run:
+          name: Install env dependencies
+          command: |
+            sudo apt-get update
+            sudo apt-get install postgresql-client cmake pkg-config
```
Comment (jbarber) [inline .circleci/config.yml]: Does circlemator need these?

Comment (kruszczynski) [reply .circleci/config.yml]: Yup:
- `postgresql-client` is required by `pg` gem
- `cmake` and `pkg-config` are required by `rugged`

We also need to change the `required` check before merging this

Comment (jbarber) [reply .circleci/config.yml]: What's using postgresql though? :thinking:

Comment (jbarber) [reply .circleci/config.yml]: I've removed the required check. Ping me on slack when it's merged and I'll enable the new check.

**Pull request merged**